### ISSUE TITLE
Install all packages from repository

### ISSUE THREAD

**Issue by antoniotvr** [dslackw/slpkg#101]
Hi.

How can I install all packages from a repository?

I need to install mate (from msb repository) or kde5 (from ktown repository) but in man pages I have not found any indication on how to install all packages.

I have tried to "slpkg -s msb *" but it doesn't work.

Best regards

**Comment by dslackw** [comment]
Hi @antoniotvr,
You can try ```slpkg -s msb ""``` but you make sure for the sequence first.

**Comment by antoniotvr** [comment]
Hi.

Thank you. But in the case offre the entire repository, how Can I specify the sequenze?

Regards

**Comment by dslackw** [comment]
Try to install first all the dependencies with command ```slpkg -s msb <dep1> <dep2> <dep3> <...>``` and after you can run ```slpkg -s msb""```. Read the repository instructions.

**Comment by dslackw** [comment]
I tested in my VM Box and it works fine. I used only a command ```slpkg -s msb ""```
https://imgur.com/a/g6oxG

**Comment by antoniotvr** [comment]
Fine, thank you!

I will test It tomorrow.

Antonio



Il 23 ott 2017 8:17 PM, "Dimitris Zlatanidis" <notifications@github.com> ha
scritto:

I tested in my VM Box and it works fine. I used only a command slpkg -s msb
""
https://imgur.com/a/g6oxG

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/dslackw/slpkg/issues/101#issuecomment-338750030>, or mute
the thread
